src/Sulu/Bundle/AdminBundle/Resources/js/views/List/tests/List.test.js

```diff
@@ -1106,6 +1107,35 @@ test('Should pass router attributes array from router to the ListStore', () => {
     expect(listStore.options.title).toEqual('Sulu is awesome');
 });
 
+test('Should pass router attributes array from router to the ListStore metadataOptions', () => {
+    const List = require('../List').default;
+    const router = {
+        bind: jest.fn(),
+        attributes: {
+            id: '123-123-123',
+            locale: 'en',
+            title: 'Sulu is awesome',
+        },
+        route: {
+            options: {
+                adapters: ['table'],
+                apiOptions: {},
+                listKey: 'test',
+                locales: ['en', 'de'],
+                resourceKey: 'test',
+                routerAttributesToListMetadata: ['locale', 'title', 'id'],
```

Comment:
Maybe leave one out here, so that we are sure that not just all properties are copied.